src/main/java/org/gridsuite/network/map/NetworkMapService.java

```diff
@@ -1442,6 +1463,23 @@ public List<LineMapData> getMapLines(UUID networkUuid, String variantId, List<St
         }
     }
 
+    public List<HvdcLineMapData> getMapHvdcLines(UUID networkUuid, String variantId, List<String> substationsIds) {
+        Network network = getNetwork(networkUuid, getPreloadingStrategy(substationsIds), variantId);
+
+        if (substationsIds == null) {
+            return network.getHvdcLineStream().map(NetworkMapService::toBasicMapData).collect(Collectors.toList());
+        } else {
+            return substationsIds.stream()
+                    .flatMap(substationsId -> network.getSubstation(substationsId).getVoltageLevelStream())
+                    .flatMap(voltageLevel -> voltageLevel.getConnectableStream(HvdcConverterStation.class))
+                    .map(HvdcConverterStation::getHvdcLine)
+                    .filter(Objects::nonNull)
+                    .map(NetworkMapService::toBasicMapData)
+                    .distinct()
```

Comment:
why distinct on the dto ?

Reply:
Because we get the HvdcLines from the HvdcConverterStations and since we have a line between 2 HvdcConverterStations, we will have 2 HVDCLines

Reply:
No I mean why on the dto and not on the iidm objects ? But it's ok it's the same as in getLines() so it's good even if not perfect. 
Also in getLines() we use a set explicitly but here we use .distinct(), but that's also ok even if not perfect.